pkg/genericapiserver/genericapiserver.go

```diff
@@ -80,7 +80,6 @@ func NewStorageDestinations() StorageDestinations {
 
 // AddAPIGroup replaces 'group' if it's already registered.
 func (s *StorageDestinations) AddAPIGroup(group string, defaultStorage storage.Interface) {
-	glog.Infof("Adding storage destination for group %v", group)
```

Comment:
Sorry, that was my mistake - I thought I added that line, and was removing my mess.